src/feature-viewer.js

```diff
@@ -1165,7 +1223,54 @@ function createFeature(sequence, div, options) {
 //                    .style("shape-rendering", "crispEdges")
                     .call(d3.helper.tooltip(object));
                 })
-                
+                forcePropagation(histog);
+            },
+            bar: function (object, position) {
+                if (object.fill === undefined) object.fill = true;
+                var histog = svgContainer.append("g")
+                    .attr("class", "bar")
+                    .attr("transform", "translate(0," + position + ")");
+                var dataline=[];
+                dataline.push([{
```

Comment:
Why is this definition of dataline[]? Shouldn't it be passed in the feature deifinition?

Reply:
This is related to the X axis required data is passed in later. The implementation is similar to the line chart type X axis.